mmv1/products/bigquerybiglake/Catalog.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Resource
+name: "Catalog"
+description: |
+  'Catalogs are top-level containers for Databases and Tables.'
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    "Manage open source metadata with BigLake Metastore": "https://cloud.google.com/bigquery/docs/manage-open-source-metadata#create_catalogs"
+  api: "https://cloud.google.com/bigquery/docs/reference/biglake/rest/v1/projects.locations.catalogs"
+base_url: "projects/{{project}}/locations/{{location}}/catalogs"
+self_link: "projects/{{project}}/locations/{{location}}/catalogs/{{catalog_id}}"
+immutable: true
+create_url: "projects/{{project}}/locations/{{location}}/catalogs?catalogId={{catalog_id}}"
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "bigquery_biglake_catalog"
+    primary_resource_id: "default"
+    vars:
+      catalog_id: "my_catalog"
+      location: "US"
+properties:
+  - !ruby/object:Api::Type::String
+    name: "catalog_id"
```

Comment:
use `name` as the field name for the main id instead to benefit from default behavior

Reply:
```suggestion
    name: "name"
```

Reply:
should be marked immutable like in the docs - the API will return the longer version of the name but we suppress the diff as long as the last part of the long name is the same.